### PR TITLE
add input_category and output_category and qualifiers to TestCase class

### PR DESCRIPTION
**Pull request by sierra-moxon**
- a change to the runner_setting variable to take into account Richard's changes.  
- refreshed the spreadsheet dump (and added identifiers where I could find them in the missing output_id field using OLS).
- add input_category and output_category and qualifiers to TestCase

Note: also added some additional suites with names in the Test Run Schedule and the (rough - e.g. asked for 5, gave you 5, asked for ~20, gave 27) number of cases per suite as defined.

### CODE REVIEW

File: test_cases/TestCase_11.json [hunk 1]
```diff
@@ -149,15 +109,29 @@
...
+            "parameter": "biolink_object_aspect_qualifier",
+            "value": ""
+        },
+        {
+            "parameter": "biolink_object_direction_qualifier",
+            "value": ""
+        }
+    ],
+    "input_category": "biolink:Disease",
+    "output_category": "biolink:ChemicalEntity",
```
Comment (maximusunc) [inline test_cases/TestCase_11.json:132]: Can it always be expected that the output category be the same across test case assets? I see it as adding specificity for the query template used, but would there ever be a case where the outputs are different categories? I think I could imagine a test template that ends up sending NamedThing as the output category?

File: test_suites/test_suite_output.json
Comment (maximusunc) [inline test_suites/test_suite_output.json:1]: I would like to consider the naming convention of the test suite files. My preference is that the names either correspond to the names given in the Test Run Schedule doc, or we number them just like the TestCases and TestAssets, and their `name` field matches a Test Run name.

Comment (sierra-moxon) [reply test_suites/test_suite_output.json:1]: changes made.

File: test_cases/TestCase_11.json (hunk 1 above)
Comment (sierra-moxon) [reply test_cases/TestCase_11.json:132]: That would definitely change the way we generate the TestCases.  This PR doesn't restrict us from adding another way of making TestCases where the TestCase input/output categories don't match the TestAsset input/output categories.

**Review by maximusunc — CHANGES_REQUESTED** [review]
Ok, so looking at comments made on the Test Run Schedule, I think these changes need to be made to the Test Suites:
- semantic_delta_and_time_profiling_prod should just be the single Benchmark Test Case.
- stress_test_prod needs to point to CI instead.

**Review by maximusunc — APPROVED** [review]
Looks good!

**Pull request merged**